test/features/discord.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Discord Module
+
+  Scenario: Get List of Users
+    When make a GET Request to "/Discord"
```

Comment:
I think this should be a `given`, for example

`Given a GET request is made to "/discord"

Note: I think `request` and `discord` should be lowercase

Reply:
From the [docs](https://cucumber.io/docs/gherkin/reference/#example)

What I understand is:
**Given** is used to describe the initial context of the system, we should avoid user interaction here
**When** is used to describe an event or an action.
**Then** is used to describe an expected outcome, or result.

This `make a GET request to "/discord"` is a user action, I think it's best to have it in `When`
For Given we can have
`Given There is a (GET) API Endpoint "/discord"`